### ISSUE TITLE
Feature Request: support `.prettierrc` config file

### ISSUE THREAD

**Issue by farfromrefug** [inferrinizzard/prettier-sql#61]
It would be very useful and help github repos maintenance if the tool supported `.prettierrc` config file to pass config options. 
It the cli did support this then the vscode extension would support it to

**Comment by inferrinizzard** [comment]
Yes, the future plan is for this module to eventually be able to interoperate as a Prettier Plugin but that is still in the planning phase.
